bin/resin-preload

```diff
@@ -132,6 +131,12 @@ buildOutputStream.pipe(process.stdout)
 
 streamToPromise(buildOutputStream)
 .then(() => {
+  if (options.apiToken) {
+    return resin.token.set(options.apiToken)
```

Comment:
Fixed, I used a temporary directory for all cases.
If someone wants to use the resin-cli token, he can use `resin-cli preload`.